src/goto-instrument/cover.h

```diff
@@ -46,18 +45,26 @@ struct cover_configt
 
 void instrument_cover_goals(
   const symbol_tablet &,
+  const cover_configt &,
   goto_functionst &,
   coverage_criteriont,
   message_handlert &message_handler);
 
 void instrument_cover_goals(
   const symbol_tablet &,
+  const cover_configt &,
   goto_programt &,
   coverage_criteriont,
   message_handlert &message_handler);
 
+std::unique_ptr<cover_configt>
+get_cover_config(const optionst &, const symbol_tablet &, message_handlert &);
```

Comment:
Should be documented, as should the overload with more parameters.

Actually, the different overloads seem to be kind of doing different things, so maybe they should have different names? One seems to create a basic `cover_configt` the other just adds a bunch of filters? Maybe it should just take the `cover_config` by ref and modify it?

Reply:
This is documented in the cpp file, it was the way this was done before I edited the files.

I'd keep the same names, the overload with extra parameters does strictly more than the overload without it, and I believe this is a standard characteristics of overloads.